openbb_platform/README.md

```diff
@@ -74,11 +74,10 @@ Set your keys at [OpenBB Hub](https://my.openbb.co/app/sdk/api-keys) and get you
 >>> from openbb import obb
 >>> obb.user.credentials.fmp_api_key = "REPLACE_ME"
 >>> obb.user.credentials.polygon_api_key = "REPLACE_ME"
-
->>> # Persist changes in ~/.openbb_platform/user_settings.json
->>> obb.account.save()
 ```
 
+> To persist changes in OpenBB Hub, run `obb.account.save()` after the login.
+
```

Comment:
I made this change for cosmetic reasons. Feel free to revert this if it doesn't make sense!

Reply:
There's a similar comment below, can you change for consistency pls?